lib/hawkular/hawkular_client.rb

```diff
@@ -66,7 +66,7 @@ def tokens
 
     # this is in a dedicated method, because constructor opens the websocket connection to make the handshake
     def init_operations_client
-      Operations::OperationsClient.new(entrypoint: @state[:entrypoint],
+      Operations::Client.new(entrypoint: @state[:entrypoint],
```

Comment:
What do you think about also make the arguments uniform? We could have a new method like `#create` so we not break compatibility.